### DISCUSSION TITLE
Make it possible to rename issues directly on a board

### DISCUSSION THREAD

**Discussion by kamstrup** [community/community#4815] [Projects and Issues]
When reviewing tasks in a weekly iteration we often end up renaming some issues to clarify the intent. Everything flows a bit smoother when it's possible to rename the issue directly without leaving the board.

**Comment by knyghty** [comment]
FWIW, you can do this on a table, just not on a board. So if you have some bulk renaming to do, you can switch the view and switch back. A bit annoying though.
